packages/mermaid/src/diagrams/flowchart/parser/flow-style.spec.js

```diff
@@ -286,6 +286,14 @@ describe('[Style] when parsing', () => {
     expect(edges[0].type).toBe('arrow_point');
   });
 
+  it('should handle style definitions within number of edges', function () {
+    const res = flow.parser.parse('graph TD\n' + 'A-->B\n' + 'linkStyle 0 stroke-width:1px;');
```

Comment:
Let's use a template string here instead

Reply:
Actually I followed the convention that was present in previous code. However I have converted it into template string.